README.md

```diff
@@ -24,16 +24,30 @@ what configuration you need on the AWS side.
 
 ### General recommendations
 
-The number of `replicas` in the [default values file](helm/loki/values.yaml) are generally considered as safe.
+The number of `replicas` in the [default values file](https://github.com/giantswarm/loki-app/helm/loki/values.yaml) are generally considered as safe.
 If you reduce the number of `replicas` below the default recommended values, expect undefined behaviour and problems.
 
+#### Prepare config file
+
+1. Create app config file
+    Grab the included [sample config file](https://github.com/giantswarm/loki-app/sample_configs/values-gs.yaml) or [azure sample config file](https://github.com/giantswarm/loki-app/sample_configs/values-gs-azure.yaml) ,
+    read the comments for options and adjust to your needs. To check all available
+    options, please consult the [upstream `values.yaml` file](https://github.com/giantswarm/loki-app/helm/loki/values.yaml).
```

Comment:
Should this link actually point to the upstream repo ?

```suggestion
    options, please consult the [upstream `values.yaml` file](https://github.com/grafana/helm-charts/blob/main/charts/loki/values.yaml).
```

Reply:
You're right, that's ambiguous. :thinking: 
The grafana-upstream one will be the most recent version, which may not be ours. So it's "our copy of the upstream file".
Thanks for pointing out, I'll fix the wording!